nixos/modules/programs/mouse-actions.nix

```
@@ -3,13 +3,39 @@
 let
   cfg = config.programs.mouse-actions;
 in
-  {
-    options.programs.mouse-actions = {
-      enable = lib.mkEnableOption ''
-        mouse-actions udev rules. This is a prerequisite for using mouse-actions without being root.
+{
+  options.programs.mouse-actions = {
+    enable = lib.mkEnableOption ''
+      mouse-actions udev rules. This is a prerequisite for using mouse-actions without being root.
+    '';
```

This should not say that it specifically adds the udev rules anymore. Maybe simply

```suggestion
    enable = lib.mkEnableOption "mouse-actions";
```